src/storage/gcloud-storage/gcp-storage.service.ts

```diff
@@ -110,35 +118,47 @@ export class GCPStorageService {
         return ok(true);
     }
 
-    async verifyImageUrl(url: string): Promise<boolean> {
+    async verifyImageUrl(
+        url: string,
```

Comment:
Would be great if this was a `URL` - then we can do things like `url.host === expectedHost`